doc-content/enterprise-only/bpmn/proc-create-process-data-object.adoc

```diff
@@ -23,7 +23,12 @@ In {PRODUCT} {PRODUCT_VERSION_LONG}, limited support for data objects is provide
 |Description
 
 | *Name*
-| The name of the data object. You can also double-click the data object shape to edit the name. 
+| The name of the data object. You can also double-click the data object shape to edit the name.
+
+| *Metadata Attributes*
+a| Add any custom metadata attribute name and value that you want to use for custom event listeners, such as a listener to implement some action when a metadata attribute is present.
+
+The *Metadata Attributes* enable the new `metaData` extensions to BPMN diagrams and modify the behavior of the overall data object.
```

Comment:
"modify the behavior of the overall data object" seems unclear to me. Do these attributes matter only with custom listeners? Or do they actually affect "behaviour of the object"? If the latter, then in what way do they affect this behaviour?

(Will not repeat this note for the same phrase elsewhere)

Reply:
@mramendi Let's say the following keys and values are added to the metadata attribute for a task:
![image](https://user-images.githubusercontent.com/59050394/130039166-86f95d70-510c-49e7-b682-3d370c3d59c4.png)

Then using metadata attribute, it is identified whether the task's attribute is defined or it is needed to be used in conjunction with a custom event listener.